comp/core/tagger/types/types.go

```diff
@@ -22,6 +22,25 @@ type Entity struct {
 	hash string
 }
 
+// ProductOrigin is the origin of the product that sent the entity.
+type ProductOrigin string
+
+const (
+	// ProductOriginDogStatsD is the ProductOrigin for DogStatsD.
+	ProductOriginDogStatsD ProductOrigin = "dogstatsd"
```

Comment:
I think using a [iota enum](https://yourbasic.org/golang/iota/) could be a good idea here.

Reply:
Done